tests/para-tests/moonbase/test-runtime-upgrade.ts

```diff
@@ -85,27 +84,18 @@ describeParachain(
         // ~200000 + init 60000 + error marging 140000
         this.timeout(400000);
 
-        const currentVersion = await (
-          (await context.polkadotApiParaone.query.system.lastRuntimeUpgrade()) as any
-        ).unwrap();
-        expect(currentVersion.toJSON()).to.deep.equal({
-          specVersion: Number(baseRuntime),
-          specName: "moonbase",
-        });
+        const currentVersion = await context.polkadotApiParaone.rpc.state.getRuntimeVersion();
+        expect(currentVersion.specVersion.toJSON()).to.equal(Number(baseRuntime));
         console.log(
           `Current runtime: ✅ runtime ${currentVersion.specName.toString()} ` +
             `${currentVersion.specVersion.toString()}`
         );
 
         await context.upgradeRuntime({ runtimeName: "moonbase", runtimeTag: RUNTIME_VERSION });
 
-        process.stdout.write(`Checking on-chain runtime version ${localVersion}...`);
         expect(
-          await (await context.polkadotApiParaone.query.system.lastRuntimeUpgrade()).toJSON()
-        ).to.deep.equal({
-          specVersion: Number(localVersion),
-          specName: "moonbase",
-        });
+          (await context.polkadotApiParaone.rpc.state.getRuntimeVersion()).specVersion.toJSON()
```

Comment:
This should be a `....toNumber()`

Reply:
`toJSON()` is also converting to `Number()`